server/ai_mediaserver.go

```diff
@@ -458,6 +461,7 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 			mediaMTXInputURL = mediaMTXRtmpURL
 		}
 		mediaMTXOutputURL := mediaMTXInputURL + "-out"
+		mediaMTXOutputAlias := fmt.Sprintf("%s-%s-out", mediaMTXInputURL, requestID)
```

Comment:
So, we send the output RTMP to both `mediaMTXOutputURL` and `mediaMTXOutputAlias`. Right?

Then, 2 questions:
1. AFAIU Frontend knows only about `mediaMTXOutputURL`, then who will use `mediaMTXOutputAlias`?
2. How does it help with the rapid reconnect? I think the RTMP into `mediaMTXOutputURL` will fail anyway.